Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -343,10 +340,6 @@ public final class EpoxySwiftUIHostingView<RootView: View>: UIView, EpoxyableVie
 
     addSubview(viewController.view)
 
-    // Get the view controller's view to be sized correctly so that we don't have to wait for
-    // autolayout to perform a pass to do so.
-    viewController.view.frame = bounds
-
```

Comment:
was this causing an issue too?

Reply:
This wasn't but I'd seen in the original pr [here](https://github.com/airbnb/epoxy-ios/pull/77) that we were updating this bit of code and it seemed to have been introduced for working the `._render` method hence I removed it. I could update it back but didn't see any issues in the example project that @erichoracek had originally tested with. I could hide this behind a `CollectionViewConfiguration` flag also?

Reply:
Discussed with Michael offline and we agreed on leaving this in since we do this in our cells as well to avoid redundant SwiftUI view sizing